client/my-sites/invites/invite-accept-logged-in/style.scss

```diff
@@ -1,6 +1,5 @@
 .invite-accept-logged-in__join-as {
 	color: var( --color-neutral-60 );
-	font-family: $sans;
```

Comment:
Is this one intentional (being `$sans` instead of `$serif`)?

Reply:
IIRC, the reason it was explicitly declared as `$sans` is because it was nested within a `$serif`, so removing the serif on the parent also removes the need for re-defining the font family on the child element.